torch/_utils_internal.py

```diff
@@ -356,7 +356,5 @@ def maybe_upload_prof_stats_to_manifold(profile_path: str) -> Optional[str]:
     return None
 
 
-def log_chromium_event_internal(
-    event, stack, compile_id, logger_uuid, start_timestamp=None
-):
+def log_chromium_event_internal(event, stack, compile_id, logger_uuid, start_time_ns):
```

Comment:
Because the implementation isn't visible from OSS, it would be really helpful to have accurate type annotations here